src/components/ui/GoToRepoButton/GoToRepoButton.tsx

```diff
@@ -0,0 +1,16 @@
+import GitHubIcon from '@assets/svg/github-logo.svg'
```

Comment:
This file should go in the `components/common` since it's a derivative of a UI component.

Reply:
Thanks for the catch! Working on it :+1: